Sources/TokamakCore/Fiber/FiberReconciler.swift

```diff
@@ -270,6 +270,14 @@ public final class FiberReconciler<Renderer: FiberRenderer> {
     self.alternate = current
     current = alternate
 
+    print("""
+      reconcile done
+      mutations were:
+      \(visitor.mutations.map { "  \($0)" }.joined(separator: "\n"))
+      alternate is \(self.alternate.recursiveDescription)
+      current is \(current.recursiveDescription)
+      """)
```

Comment:
Can you clean up any print statements before merging? Better debug logging could be tackled in a separate PR.